validation/tests/test_recordings_search.py

```diff
@@ -59,7 +59,7 @@ def test_search_recordings(client, bake_recording):
     assert isinstance(recordings, list)
     assert len(recordings) == 1
     recording = recordings[0]
-    assert recording.get("wordform") == phrase.transcription
+    assert recording.get("wordform") == phrase.field_transcription
```

Comment:
Hmm... I'm unsure about this one. So, this is when itwêwina asks the speech-db for recordings for, e.g., "wâpamêw". Although this is (maybe?) a breaking change, I think we want it to search for the _validated_ transcription rather than the original transcription!

```suggestion
    assert recording.get("wordform") == phrase.transcription
```

Basically: when itwêwina wants a recording for a word (and you can assume itwêwina has "validated" wordforms), what should it match in the database? This same comment goes for the rest of the tests in this file.